src/conntrack/message.rs

```diff
@@ -0,0 +1,72 @@
+// SPDX-License-Identifier: MIT
+
+use crate::{
+    buffer::NetfilterBuffer,
+    conntrack::nlas::nla::ConntrackNla,
+    constants::{IPCTNL_MSG_CT_GET, NFNL_SUBSYS_CTNETLINK},
+};
+use netlink_packet_core::{
+    DecodeError, DefaultNla, Emitable, Parseable, ParseableParametrized,
+};
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+#[non_exhaustive]
+pub enum ConntrackMessage {
+    Get(Vec<ConntrackNla>),
+    Other {
+        message_type: u8,
+        nlas: Vec<DefaultNla>,
+    },
+}
+
+impl ConntrackMessage {
+    pub(crate) const SUBSYS: u8 = NFNL_SUBSYS_CTNETLINK;
+
+    pub fn message_type(&self) -> u8 {
```

Comment:
Can we introduce a enum for this `message_type`?

Like `ContrackMessageType::Get` and etc.